README.md

```diff
@@ -1,6 +1,6 @@
 ## Overview
 
-Synopsys Detect is Black Duck's intelligent scan client that scans code bases in your projects and folders to perform compositional analysis. Synopsys Detect sends scan results to Black Duck, which generates risk analysis when identifying open source components, licenses, and security vulnerabilities.
+Synopsys Detect scans code bases in your projects and folders to perform compositional analysis and functions as a Black Duck intelligent scan client. Synopsys Detect can send scan results to Black Duck, which generates risk analysis when identifying open source components, licenses, and security vulnerabilities.
```

Comment:
How about "as an intelligent scan client for Black Duck and other Synopsys SCA products"?

Reply:
Thank you. I'll make the tweak. (I don't know if the README can use variables, so will have to leave the coded name.)